profiles/package.mask

```diff
@@ -29,6 +29,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Aric Belsito <lluixhi@gmail.com> (13 Feb 2018)
+# Package name change from msgpack-python to msgpack breaks dependencies.
+=dev-python/msgpack-0.5.4
```

Comment:
`>=`? I suppose the same will affect future bumps.

Reply:
True. I'll change that.